specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -1437,7 +1437,7 @@
             }
           }
         },
-        "x-ms-odata": "#/definitions/BookmarkRelation",
+        "x-ms-odata": "#/definitions/Relation",
```

Comment:
Line: 6921 - In the definition of 'relation', 'relatedResourceName' should be readonly as 'relatedResourceId' is the only one required.